server/db/db.js

```diff
@@ -364,14 +364,11 @@ async function getRecentButtonsVitals(pgClient) {
     const results = await helpers.runQuery(
       'getRecentButtonsVitals',
       `
-      SELECT a.*
-      FROM (
-        SELECT DISTINCT ON (b.display_name) bv.*
-        FROM buttons_vitals AS bv
-        LEFT JOIN buttons AS b ON b.id = bv.button_id
-        ORDER BY b.display_name, bv.created_at DESC
-      ) AS a
-      ORDER BY a.created_at
+      SELECT b.id as button_id, bv.id, bv.battery_level, bv.created_at
```

Comment:
Needed to use `b.id` instead of `bv.button_id` because in the cases where there has never been a Button heartbeat, the `bv.button_id` value will be `null`.